SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -9,7 +9,7 @@
 Summary:        Linux Kernel for HCI
 Name:           kernel-hci
 Version:        5.15.102.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Please also bump the release number in ./SPECS-SIGNED/kernel-hci-signed/kernel-hci-signed.spec